src/main/scala/wdl4s/cwl/CommandLineTool.scala

```diff
@@ -44,18 +42,18 @@ case class CommandInputEnumSchema(
 
 case class CommandInputArraySchema(
                                     items:
-                                      CwlType :+:
+                                    CwlType :+:
```

Comment:
Yeah - not sure if this is a war of autoformatters or not but I'd prefer to not see these going back and forth every PR :)